migrations/20181012053803-create-bookmark.js

```diff
@@ -0,0 +1,36 @@
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Bookmarks', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    userId: {
+      type: Sequelize.INTEGER,
+      onDelete: 'CASCADE',
+      reference: {
+        id: 'id',
+        model: 'Users'
+      }
+    },
+    articleId: {
+      type: Sequelize.INTEGER,
+      onDelete: 'CASCADE',
+      reference: {
```

Comment:
To add foreign key constraints to a model use references instead of reference.

Reply:
Okay, I will look into it